packages/fluentui/docs/src/examples/components/Menu/Visual/MenuExampleCompose.shorthand.tsx

```diff
@@ -29,24 +32,53 @@ const MenuitemIndicatorSaturated = compose(MenuItemIndicator, {
 
 const MenuItemBlue = compose(MenuItem, {
   displayName: 'MenuItemBlue',
+
   slots: {
     icon: MenuItemIconGreen,
-    content: MenuItemContentPink,
+
+    content: MenuItemContentPurple,
+
     indicator: MenuitemIndicatorSaturated,
+
     wrapper: MenuItemWrapperDashed,
   },
 });
 
+const MenuCoral = compose<'ul', MenuProps, MenuStylesProps, MenuProps, MenuStylesProps>(Menu, {
```

Comment:
@mnajdova, @layershifter does the consumer need to know that Menu renders `ul`? Should not we expose it and have something like `Menu.defaultElementType`?

Reply:
It's questionable, I don't think so